### DISCUSSION TITLE
Github Pull Request : Branch drop down searchbox filter is disabled

### DISCUSSION THREAD

**Discussion by rajxd** [community/community#129272] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hello, When I logged in today, and trying to give PR, previously I was able to filter my branches by typing into the search box, but now it is disabled. How to enabled that?

![image](https://github.com/community/community/assets/18751312/511e3a39-174c-4fa4-aab2-a7298f90bfd1)

**Comment by asishkumar-gouda** [comment]
Hii @rajxd can you please elaborate about the issue more adding more screenshots might help.

**Comment by asishkumar-gouda** [comment]
Hii @rajxd I have checked the issue this is the issue with everyone in GitHub that the search bar is removed so no worries they will bring it back soon. As there is no release notes available for this update !!

You can you Github Desktop or GITHUBCLI if possible.

Thanks if this helps!!

**Comment by rajxd** [comment]
@ASISHGOUDA 

You can find search box in below image.
![Screenshot (290)](https://github.com/community/community/assets/18751312/13a6d0a5-9aaf-4e3b-8b82-27ba1b4bd6f5)


But now it is disappeared while creating Pull Request.

**Reply by asishkumar-gouda** [reply, marked as answer]
Thats the Bug everyone is been facing by this morning I have checked there is no release notes for the same hope this gets fixed ASAP by github as the search box feature is the vital one for many devs and PM's and couldn't be ignored.

Hope this might help.

Thankyou

**Reply by rajxd** [reply]
This looks resolved now.

**Comment by ans-naveed** [comment]
Same problem here.

**Comment by jengerph** [comment]
I am using the Github Desktop workaround but I do not get the change visibility that the web provides.  Hopefully this will be fixed by Monday!

**Comment by umerdotcom** [comment]
Same problem here!

**Comment by swatantrakumar** [comment]
Same Problem!!!

**Comment by birdkiwi** [comment]
Dear GitHub, please, revert back this change. :pray:

**Comment by kwebdev11** [comment]
Yes we are facing the same issue, Search bar is disable when click on baseBranch to change it, there are lot of remote branches , its hard to find the selected one due to disable of Searchbar.

![image](https://github.com/community/community/assets/87276852/77ae1735-112c-410b-a6d3-e0e17a575b3c)

**Comment by xenera-songtao** [comment]
Same problem, very bad.

**Comment by qlimited** [comment]
Same Problem!!!

**Comment by YichuanSun** [comment]
Me too!!!

**Comment by ritumcu** [comment]
For time being, can try using this pattern if you know the branch names - https://github.com/aws-samples/aws-serverless-workshops/compare/TARGET-BRANCH...YOUR-BRANCH?expand=1
Change the path accordingly to your repo.

**Comment by zhengjitf** [comment]
Just share a quick fix, I use it on tampermonkey script (Run at: document-end):

```js
// ==UserScript==
// @name         New Userscript
// @namespace    http://tampermonkey.net/
// @version      2024-06-21
// @description  try to take over the world!
// @author       You
// @match        https://github.com/*
// @icon         https://www.google.com/s2/favicons?sz=64&domain=github.com
// @grant        none
// ==/UserScript==

(function() {
    'use strict';

    for (const searchInput of document.querySelectorAll('.SelectMenu-filter')) {
        if (
            searchInput.parentElement?.getAttribute('slot') !== 'tablist-wrapper' ||
            searchInput.getAttribute('slot') !== 'after-tabs'
        ) {
            const tablistWrapper = searchInput.parentElement?.querySelector(`[slot="tablist-wrapper"]`)
            searchInput.setAttribute('slot', 'after-tabs')
            tablistWrapper?.appendChild(searchInput)
        }
    }
})();
```

**Reply by emil-peters** [reply]
Thanks, works like charm :raised_hands:

**Reply by harry-hathorn** [reply]
madness

**Comment by harry-hathorn** [comment]
Same problem here

**Comment by shittinun** [comment]
Same problem

**Comment by AlbertoESoler** [comment]
same problem. Fix please!!
